nx/include/switch/services/hidsys.h

```diff
@@ -0,0 +1,20 @@
+/**
+ * @file hidsys.h
+ * @brief hid:sys service IPC wrapper.
+ * @author exelix
+ */
+#pragma once
+#include "../types.h"
+#include "../kernel/event.h"
+#include "../services/sm.h"
+
+Result hidSysInitialize(void);
+
+void hidSysExit(void);
+
+Result hidSysEnableAppletToGetInput(bool enable);
+
+/**
+* @brief Returns an event that fires when the home button is pressed, this will prevent the home menu from opening when the button is pressed.
```

Comment:
"this will prevent ..." Was this really intended for this cmd?

Reply:
I think so, libappletRequestHomeMenu still works as expected and when closing the event the button starts behaving normally again.